tensorflow/lite/kernels/fully_connected.cc

```diff
@@ -64,6 +64,49 @@ constexpr int kBiasTensor = 2;
 constexpr int kOutputTensor = 0;
 constexpr int kShuffledInputWorkspaceTensor = 1;
 
+inline TfLiteStatus CheckTypes(TfLiteContext* context,
+                               const TfLiteTensor* input,
+                               const TfLiteTensor* filter,
+                               const TfLiteTensor* bias, TfLiteTensor* output,
+                               TfLiteFullyConnectedParams* params) {
+  const bool is_Quantized =
+      ((filter->type == kTfLiteUInt8) || (filter->type == kTfLiteInt8));
+  const bool is_Hybrid = is_Quantized && (input->type == kTfLiteFloat32);
```

Comment:
Thanks, can you make the variable name style is_quantized instead of is_Quantized here and for all the other variables.